apache-php/README.md

```diff
@@ -16,23 +15,21 @@ The environment come with  some PHP extensions: apcu, mcrypt, intl, mysql, curl,
 You can directly pull this image from [Docker hub](https://hub.docker.com/r/akeneo/apache-php/) by running:
 
 ```bash
-$ docker run --name apache-php -p 8080:80 -d akeneo/apache-php:php-5.6
+$ docker run -i -t --rm akeneo/apache-php:php-5.6 bash
 ```
 
-Access the URL `localhost:8080` with your web browser to check that the container works.
-
 ### From GitHub
 
 Clone the repository, go inside the created folder, and build the docker image:
 
 ```bash
-$ docker build -t "apache-php" .
+$ docker build -t "php" .
 ```
 
 Then you can run a container like this:
 
 ```bash
-$ docker run --name apache-php -p 8080:80 -d apache-php
+$ docker run -i -t --rm php bash
```

Comment:
"akeneo-php" could avoid misunderstanding between real command name "php" and name of the docker. I mean for docker newbies.